messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/engine/EngineWebView.kt

```diff
@@ -36,6 +38,19 @@ internal class EngineWebView @JvmOverloads constructor(
         } catch (e: Exception) {
             Log.e(GIST_TAG, "Error while creating EngineWebView: ${e.message}")
         }
+        // Get WebViewClientInterceptor from GistSdk directly
+        engineWebViewClientInterceptor = GistSdk.engineWebViewClientInterceptor
```

Comment:
Instead of a strong reference being held in this class, would it be better memory management to change the class property to a getter that accesses the singleton each time requested? 

``kt
private var engineWebViewClientInterceptor: EngineWebViewClientInterceptor?
  get() = GistSdk.engineWebViewClientInterceptor
```

If we need a strong reference for a specific reason, a comment added would be helpful to make sure we always do that.